### PR TITLE
Pollutant dropdown

### PR DESCRIPTION
**Pull request by paigewilliams**
TODO:
- [X] might make a custom hook for getting the layers since we are doing it in the pollutant select and layers component
- [X] code cleanup 
- [X] fix initial load to only show the default pollutant

### CODE REVIEW

**Comment by pricealexandra** [comment]
this is looking so awesome!

File: components/pollutantSelect/pollutantSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          groupIds: layers.map(({ id }) => id),
+        });
+      }
+
+      return accum;
+    }, []);
+  }, [feltLayers.data]);
+
+  const handlePollutantChange = useCallback(
+    (_value: string, option: Pollutant) => {
```
Comment (paigewilliams) [inline components/pollutantSelect/pollutantSelect.tsx:61]: I like how we can pass additional data to the select option to use here

File: components/pollutantSelect/pollutantSelect.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      setValue(_value);
+      const hiddenLayers = pollutantLayers
```
Comment (paigewilliams) [inline components/pollutantSelect/pollutantSelect.tsx:63]: hide layers that are not the one selected

File: components/pollutantSelect/pollutantSelect.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        .filter(({ value }) => value !== _value)
+        .map(({ groupIds }) => groupIds)
+        .flat();
+
+      felt.setLayerGroupVisibility({
```
Comment (paigewilliams) [inline components/pollutantSelect/pollutantSelect.tsx:68]: this doc was helpful for best practices for toggling layer visibility https://developers.felt.com/js-sdk/hiding-and-showing#focusing-on-a-single-layer

**Review by pricealexandra — APPROVED** [review]
this is awesome!!

**Pull request merged**